DSharpPlus/Entities/Channel/DiscordChannel.cs

```diff
@@ -84,13 +84,27 @@ public DiscordChannel Parent
         public bool IsPrivate
             => this.Type == ChannelType.Private || this.Type == ChannelType.Group;
 
+        /// <summary>
+        /// Gets whether this channel is a DiscordThreadChannel.
+        /// </summary>
+        [JsonIgnore]
+        public bool IsThread
+            => this.Type == ChannelType.NewsThread || this.Type == ChannelType.PrivateThread || this.Type == ChannelType.PublicThread;
```

Comment:
Presumptively, this will never be true because DiscordThreadChannel doesn't extend DiscordChannel, therefore a DiscordChannel can never be a thread channel.

Reply:
I'll tmp remove this field. When threads launch or when D#+ changes this implementation we can re-implement it.

Reply:
Removed.